x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/builders.ts

```diff
@@ -19,7 +19,11 @@ export const initialEndpointPageState = (): Immutable<EndpointState> => {
     loading: false,
     error: undefined,
     endpointDetails: {
-      activityLog: createUninitialisedResourceState(),
+      activityLog: {
+        page: 1,
+        pageSize: 10,
```

Comment:
Suggestion: increase the page size. I think getting 50 at a time would be an ok value. but 10 is too low IMO and would possibly generate too many HTTP requests

Reply:
Indeed. I have it at 10 to use it with dummy data. Will change it to 50.

Reply:
done 09d2156a8edda8c6a82c0264ac77dbcf58d387d5